eras/allegra/impl/src/Cardano/Ledger/Allegra/Translation.hs

```diff
@@ -38,14 +38,6 @@ import qualified Data.Map.Strict as Map
 -- being total. Do not change it!
 --------------------------------------------------------------------------------
 
--- | Return the subset of UTxO corresponding to Byron-era AVVM addresses, which
--- are to be removed on the Shelley/Allegra boundary. This set will be passed
--- _back_ to the translation functions as the UTxO, allowing these addresses to
--- be removed. This is needed because we cannot do a full scan on the UTxO at
--- this point, since it has been persisted to disk.
-shelleyToAllegraAVVMsToDelete :: NewEpochState (ShelleyEra c) -> UTxO (ShelleyEra c)
-shelleyToAllegraAVVMsToDelete = stashedAVVMAddresses
```

Comment:
huh, I thought this was still needed. :thinking:

Reply:
You are totally right, I thought no-one uses it because I couldn't find any place where it is being used, but it turns out it still didn't land in consensus master and is part of this UTxO-HD long running PR: input-output-hk/ouroboros-network#3863

I'll bring it back.

Reply:
Done